### PR TITLE
Add GCB scripts for Jupyter UI

### LINKED ISSUE #2578: Automate image builds of Jupyter UI
We need to update the jupyter-web-app's image in gcr to include the changes made in [this PR](https://github.com/kubeflow/kubeflow/pull/2490), in order to redirect to the correct URL for the notebook.

### PR DESCRIPTION
**Pull request by kimwnasptd**
for #2578, will need a follow up PR to add the post/pre submit jobs to `prow_config.yaml`

* Create a `build_image.sh`
* Create the ksonnet app for Argo

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2656)

### CODE REVIEW

File: components/jupyter-web-app/releaser/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/lib
```
Comment (jlewi) [inline components/jupyter-web-app/releaser/.gitignore]: Why are you ignoring /lib?

Comment (kimwnasptd) [reply components/jupyter-web-app/releaser/.gitignore]: I saw in one of ksonnet's [PRs](https://github.com/ksonnet/ksonnet/pull/406) that this folder is can be automatically generated. This gitignore was created from ksonnet. Should I not ignore it?

**Pull request merged**